TILApp/Views/AuthView/SignInViewController.swift

```diff
@@ -1,27 +1,77 @@
 import UIKit
+import AuthenticationServices
 
 final class SignInViewController: UIViewController {
-    private lazy var button = {
-        let button = UIButton()
-        button.setTitle("SIGN IN", for: .normal)
-        button.addTarget(self, action: #selector(signInButtonTapped), for: .touchUpInside)
-        return button
-    }()
 
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .systemBackground
 
-        view.addSubview(button)
-        button.translatesAutoresizingMaskIntoConstraints = false
+        setAppleLoginButton()
+    }
+
+    func setAppleLoginButton() {
+        let authorizationButton = ASAuthorizationAppleIDButton(type: .signIn, style: .whiteOutline)
+        authorizationButton
+            .addTarget(self, action: #selector(handleAuthorizationAppleIDButtonPress), for: .touchUpInside)
+
+        view.addSubview(authorizationButton)
+        authorizationButton.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
-            button.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            button.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            authorizationButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            authorizationButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
         ])
     }
 
-    @objc private func signInButtonTapped() {
-        let homeVC = HomeViewController()
-        navigationController?.pushViewController(homeVC, animated: true)
+    @objc func handleAuthorizationAppleIDButtonPress() {
+        let appleIDProvider = ASAuthorizationAppleIDProvider()
+        let request = appleIDProvider.createRequest()
+        request.requestedScopes = [.fullName, .email]
+
+        let authorizationController = ASAuthorizationController(authorizationRequests: [request])
+        authorizationController.delegate = self
+        authorizationController.presentationContextProvider = self
+        authorizationController.performRequests()
+    }
+}
+
+extension SignInViewController: ASAuthorizationControllerDelegate {
+    // 로그인 성공 후 동작
+    func authorizationController
+    (controller: ASAuthorizationController, didCompleteWithAuthorization authorization: ASAuthorization) {
```

Comment:
```suggestion
    func authorizationController(
        controller: ASAuthorizationController,
        didCompleteWithAuthorization authorization: ASAuthorization
    ) {
```

보통 함수 이름하고 매개변수 사이에 공백이나 줄바꿈을 넣지 않아서 이렇게 작성하면 더 좋지 않을까 싶네요.